flambe/task/script.py

```diff
@@ -66,5 +66,20 @@ def run(self) -> bool:
         runpy.run_module(self.script, run_name='__main__', alter_sys=True)
         sys.argv = sys_save
 
-        continue_ = False  # Single step, so don't continue
-        return continue_
+        _continue = False
+        return _continue
+
+    def metric(self) -> Optional[float]:
+        """Override this method to enable scheduling and searching.
+
+        This method is called every call to ``run``, and should return
+        a unique scalar representing the current performance on the
+        task, and to compare against other variants of the task.
+
+        Returns
+        -------
+        Optional[float]
+            The metric to compare different variants of your Component
+
+        """
+        return None
```

Comment:
What does it mean to return `None`?

Reply:
I was thinking we could allow returning None which means the Task cannot be used with `reduce` or certain hyperparameter search aglrotihms